data_juicer/format/formatter.py

```diff
@@ -189,14 +184,16 @@ def unify_format(
     >>>     "meta": {"date": 2012}
     >>>     'meta.src": "customized",
     >>>     "meta.version": "0.1",
-    >>>     'stats': {
+    >>>     Fields.stats: {
```

Comment:
here in this example, 'stats' should be renamed to the latest key name instead of the variable Fields.stats